airflow/utils/log/file_processor_handler.py

```diff
@@ -45,7 +45,10 @@ def __init__(self, base_log_folder, filename_template):
 
         self._cur_date = datetime.today()
         if not os.path.exists(self._get_log_directory()):
-            os.makedirs(self._get_log_directory())
+            try:
+                os.makedirs(self._get_log_directory())
+            except OSError:
```

Comment:
Please verify the `errno` if it indeed is existing, and fail otherwise. This is too broad.

Reply:
```
try:
    os.makedirs(...)
except OSError as e:
    if e.errno != errno.EEXIST:
        raise
```

Reply:
Done 😄